pkg/controller/account/ec2.go

```diff
@@ -167,6 +167,17 @@ func CreateEC2Instance(reqLogger logr.Logger, client awsclient.Client, ami strin
 			InstanceType: aws.String(awsInstanceType),
 			MinCount:     aws.Int64(1),
 			MaxCount:     aws.Int64(1),
+			TagSpecifications: []*ec2.TagSpecification{
+				{
+					ResourceType: aws.String("instance"),
+					Tags: []*ec2.Tag{
+						{
+							Key:   aws.String("Name"),
+							Value: aws.String("AaoInitRegion"),
+						},
+					},
+				},
+			},
```

Comment:
Any special AWS China req for this tag being added to EC2 instances? 
I'm asking because there's already another card/PR taking care of that: https://github.com/openshift/aws-account-operator/pull/339

Reply:
Nope, it was there for the exact reason. I will remove.